CHANGELOG.md

```diff
@@ -1,16 +1,17 @@
 # Changelog
+* Updated the **format** commnad to inlcude an update to the description file of an integration, to remove community/partner details.
```

Comment:
this did get in to 1.3.4

Reply:
it was duplicated from master (i fixed the commnad and inlcude typos in my release branch)
removed it now from changelog (it is already found in 1.3.4)